mteb/leaderboard/table.py

```diff
@@ -1,16 +1,42 @@
 from __future__ import annotations
 
+import math
+
 import gradio as gr
 import numpy as np
 import pandas as pd
 
+from mteb.models.overview import get_model_meta
 from mteb.overview import get_task
 
 
+def borda_count(scores: pd.Series) -> pd.Series:
+    n = len(scores)
+    ranks = scores.rank(method="average", ascending=False)
+    counts = n - ranks
+    return counts
```

Comment:
Yep - The only problem now is ties, which we calculate slightly differently in the paper (but I don't believe there are practically any ties), so I think it is equivalent so let us keep it as is.

Reply:
I think I checked and this piece of code is equivalent to tournament borda. Is that not what we used in the paper?

Reply:
I don't think it is equivalent in the case of ties (but there are no ties so equivalent in practice). We should just remove the tournament borda description from the paper.